.eslintrc.json

```diff
@@ -41,17 +41,6 @@
     "semi": [
       "error",
       "always"
-    ],
```

Comment:
We need this configuration. It prevents us from getting this error.
```
[eslint] The href attribute is required on an anchor. Provide a valid, navigable address as the href value. (jsx-a11y/anchor-is-valid)
(JSX attribute) to: string
```

Reply:
Okay, I see. Thanks